corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_controller.js

```diff
@@ -118,7 +118,7 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
                 },
             });
 
-            $('#select-case').click(function () {
+            $('#select-case').unbind('click').click(function () {
```

Comment:
a comment here might be useful if anyone comes across and sees this. it's kind of an odd pattern